content/html/concepts/attributes/terms/dir/dir.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
You need to delete all the extra line breaks in the metadata.

Reply:
complete